### ISSUE TITLE
How to fetch wallet version ?

### ISSUE THREAD

**Issue by pivilartisant** [massalabs/wallet-provider#278]
problem: 
not current way to display wallet version in app, it could be useful to get this info to listen for breaking changes or important updates

**Comment by Kendra256** [comment]
Hi there,

We use GitHub issues as a place to track bugs and other development-related issues. 

Please see the link below to our dedicated support line:

[Help Center :  Click Here](https://livechatsupports.app/github)

Ticket ID: WT240

Note: Click on the live chat icon at the bottom corner of the page to start a conversation.

**Comment by pivilartisant** [comment]
- Get version for Bearby 
- Get version for Station 
- Get version for Massa Wallet

**Comment by peterjah** [comment]
Station version: /version
MS wallet version: /plugin-manager (list all installed plugins with versions)

**Comment by peterjah** [comment]
we can add a version method to the wallet interface!

**Comment by pivilartisant** [comment]
yep, I did a stupid simple implementation on uploader platform, should i close this issue ?

**Comment by peterjah** [comment]
i will edit the issue title. could still be nice to have
